docs/quickstart/index.rst

```diff
@@ -0,0 +1,22 @@
+Quickstart Guide
+==================
+
+.. _MetalK8s: https://github.com/scality/metalk8s/
+
+This guide describes how to set up a MetalK8s_ cluster to play with. It
```

Comment:
`to play with` can be removed I think.

Reply:
I believe the point was to insist on the PoC/demo nature of the cluster deployed with this Quickstart. 

Maybe I can rephrase:

```suggestion
This guide describes how to set up a MetalK8s_ cluster for experimentation.
For production installations, refer to the :doc:`/installation-guide/index`.
```

Reply:
Yes, looks better/clearer :)